### DISCUSSION TITLE
onSymbolTapped not working

### DISCUSSION THREAD

**Discussion by ishafiul** [maplibre/flutter-maplibre-gl#488] [General]
It's been a week, and I'm still stuck on this issue. I don't know how to achieve this. I need to add more than 1,000 markers at the same time, and they need to be clickable. However, I can't figure out why it's not working. For a single marker, it works fine. I have created an issue for this (#487). I need help.

**Comment by Josh-Dovey** [comment]
Hi @ishafiul I've added a comment to the issue you raised [here](https://github.com/maplibre/flutter-maplibre-gl/issues/487#issuecomment-2301374897). Hope it helps!

**Reply by ishafiul** [reply]
thanks.
